common/big.go

```diff
@@ -20,13 +20,15 @@ import "math/big"
 
 // Common big integers often used
 var (
-	Big1   = big.NewInt(1)
-	Big2   = big.NewInt(2)
-	Big3   = big.NewInt(3)
-	Big0   = big.NewInt(0)
-	Big32  = big.NewInt(32)
-	Big256 = big.NewInt(256)
-	Big257 = big.NewInt(257)
+	Big1     = big.NewInt(1)
+	Big2     = big.NewInt(2)
+	Big3     = big.NewInt(3)
+	Big4     = big.NewInt(4)
+	Big0     = big.NewInt(0)
```

Comment:
Funny that `Big0` is in the middle of the list... might bump it

Reply:
Moved it up